File: complex.py
class complex_class():
    def __init__ (self,a = 0 ,b = 0):
        real,img = a,b
    def inputc(self):
            self.real = int(input("Enter real value:"))
            self.img = int(input("Enter imagnary value:"))
    def __str__(self):
        return "%s + %si"%(self.real,self.img)
    def add(self,a):
        z = complex_class()
        z.real = self.real+a.real
        z.img = self.img+a.img
        return z
    def adding(a,b):
        z = complex_class()
        z.real = a.real+b.real
        z.img = a.img+b.img
        return z
number1 = complex_class()
number2 = complex_class()
number1.inputc()
number2.inputc()
a = number1.add(number2)
print(a)
number2.inputc()
a = complex_class.adding(number1,number2)
print(a)

File: complex.java
import java.util.Scanner;
class relactiona
{   
    int num,den;
    Scanner reading = new Scanner(System.in);
    public relactiona()
    {
        num = 0 ;
        den = 1 ;
    }
    public relactiona( int x , int y)
    {
        if ( y == 0 )
        {
            System.out.println("Enter Correct data.");
            System.exit(0);
        }
        num = x ;
        den = y ;
    }
        void input()
        {
            System.out.println("Enter value of num. :: ");
            num = reading.nextInt();
            System.out.println("Enter value of den. :: ");
            den = reading.nextInt();
        }
        void output()
        {
            System.out.println( num + "/" + den );
        }
        void munt( relactiona a,relactiona b )
        {
            num = a.num*b.num;
            den = a.den*b.den;
        }
}
public class complex extends relactiona
{
    int real,img;
    Scanner reading = new Scanner(System.in);
    public complex()
    {
        real = 0 ; img = 0;
    }
    public complex( int x , int y )
    {
        real = x ; img = y;
    }
    void output()
        {
            System.out.println( real + "+" + "i" + img );
            super.output();
        }
    void input()
        {
            super.input();
            System.out.println("Enter value of Real :: ");
            real = reading.nextInt();
            System.out.println("Enter value of Img. :: ");
            img = reading.nextInt();
        }
    public static void main( String [] args )
    {
        complex obj2 = new complex();
        obj2.input();
        obj2.output();
    }
}